src/main/java/org/jabref/gui/fieldeditors/LinkedFileViewModel.java

```diff
@@ -463,7 +470,29 @@ public void download() {
         }
     }
 
+    public boolean checkSSLHandshake(URLDownload urlDownload) {
+        try {
+            urlDownload.canBeReached();
+        } catch (kong.unirest.UnirestException ex) {
+            if (ex.getCause() instanceof javax.net.ssl.SSLHandshakeException) {
+                if (dialogService.showConfirmationDialogAndWait(Localization.lang("Download file"),
+                        Localization.lang("Unable to find valid certification path to requested target(%0), download anyway?",
+                                urlDownload.getSource().toString()))) {
+                    URLDownload.bypassSSLVerification();
+                    return true;
+                } else {
+                    dialogService.showInformationDialogAndWait(Localization.lang("Download file"),
```

Comment:
I would use `notify` here, so that the user doesn't need to click "ok" again in the second dialog.

Reply:
Great, I agree with this idea!